scalableminds/webknossos
# in nml task creation, copy nml description to annotationBase

Small change, but in one of the ugliest corners… [edit: made it slightly less ugly by merging #2619 into this one]

### Mailable description of changes:
 - In NML task creation, the NML’s descriptions are copied to the task annotations.

### Steps to test:
- create task with NML that has `description` attribute set in `<experiment>` tag
- request that task, start tracing, see that the description has been copied there
- create tasks with NML that has no such field, create task without NML
- they should still work (and have no description)

### Issues:
- fixes #2616

------
